src/pages/streaming/streaming.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, ViewChild, AfterViewInit } from '@angular/core';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
+import * as Hls from 'hls.js';
+
+@IonicPage()
+@Component({
+  selector: 'page-streaming',
+  templateUrl: 'streaming.html',
+})
+export class StreamingPage implements AfterViewInit {
+  url = '';
+  hls: any;
+  @ViewChild('player') player: any;
+  constructor(
+    public navCtrl: NavController,
+    public navParams: NavParams
+  ) {}
+
+  ngAfterViewInit() {
+    this.url = this.navParams.data.url;
+    if (Hls.isSupported()) {
```

Comment:
should have a feedback if hls is not supported. For example a message (TV.NO_SUPPORT_DISCLAIMER) on screen

Reply:
![image](https://user-images.githubusercontent.com/32454870/53644509-85e70380-3c15-11e9-9262-ca4c8cb6d8a0.png)